group_vars/all.yml

```diff
@@ -131,3 +131,8 @@ install_rdbms: true
 disable_firewall: false
 
 ## End of non-modifiable variables section
+
+## host provision variables
```

Comment:
I thought I put in a coment about this, but now can't find it.  Apologies if it's duplicated.

In terms of group vars, I'd prefer if we don't have hostprovision in the name, and instead have the ability to reuse them across multiple parts of the code, now or in the future.

A suggestion for naming:
instance_ssh_user
firsttime_connect_user
control_node_key_file

Reply:
Got it, will do

Reply:
All of the suggestions here have been incorporated and tested successfully.
https://gist.github.com/jcnars/d0e4887c736ddbd3dcc5c9a26d4d24db

Pls check if it looks good.

Thanks